spec/features/super_admin/download_all_service_emails_spec.rb

```diff
@@ -0,0 +1,17 @@
+describe 'Downloading all the service emails', type: :feature do
+  let(:user) { create(:user, :super_admin) }
```

Comment:
The latest changes to the user factory means you need to explicitly pass in the trait for organisation, eg:
```
let(:user) { create(:user, :super_admin, ;with_organisation) }
```